app/Repositories/Eloquents/PostRepository.php

```diff
@@ -84,4 +84,79 @@ public function delete($id)
     {
         return $this->model()->destroy($id);
     }
+
+    // post index
+    public function getPostWithUserLimit($input, $limit = null)
+    {
+        $limit = (is_null($limit)) ? config('constants.PAGINATION_LIMIT_NUMBER',  16) : $limit;
+        $query = $this->model()->with('user')->with('tags');
+
+        return $this->search($query, $input, $limit);
+    }
+
+    // get post with paginate at post all, search module
+    public function getPostWithUserPaginate($input, $limit = null)
+    {
+        $limit = (is_null($limit)) ? config('constants.PAGINATION_LIMIT_NUMBER',  16) : $limit;
+        $query = $this->model()->with('user')->with('tags');
+
+        return $this->search($query, $input, $limit, true);
+    }
+
+    public function getTreadingPostLimit($limit = null)
+    {
+        $limit = (is_null($limit)) ? config('constants.PAGINATION_LIMIT_TREDINGPOST',  16) : $limit;
+        $query = $this->model()->with('tags')->orderBy('total_view', 'desc')->orderBy('id', 'desc')->take($limit)->get();
+
+        return $query;
+    }
+
+    public function getOwnTags()
+    {
+        $limit = config('constants.PAGINATION_LIMIT_OWN_TAG',  16);
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.Functions.FunctionCallArgumentSpacing.TooMuchSpaceAfterComma</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/PostRepository.php</br>**Line**: 116</br>Expected 1 space after comma in function call; 2 found